### PR TITLE
Restrict access to sensitive files in root

### PR DESCRIPTION
**Pull request by asarturas**

### CODE REVIEW

**Comment by andytson** [comment]
Just needs the apache fix in config-driven-helper. The configuration parameter might change to restricted_url instead of restricted_dir as part of the fix

**Comment by asarturas** [comment]
i believe fix in config-driven-helper is not a blocker for this one - we already had one file in list of directories.

**Comment by andytson** [comment]
Blocked by https://github.com/inviqa/chef-config-driven-helper/issues/47

File: attributes/default.rb [hunk 1]
```diff
@@ -66,7 +66,18 @@
...
+      '/var/',
+      '/.gitignore',
+      '/.htaccess',
+      '/.htaccess.sample',
+      '/composer.json',
+      '/cron.php',
+      '/cron.sh',
+      '/index.php.sample',
+      '/install.php',
+      '/mage',
```
Comment (andytson) [inline attributes/default.rb]: there's no mage in the magento repository. If this is just for your project you can put it in your project's project-web.json

Comment (asarturas) [inline attributes/default.rb]: It is from mage enterprise repository, see https://github.com/inviqa/magento-enterprise/blob/master/mage

Comment (andytson) [inline attributes/default.rb]: fair enough, I'd looked and didn't see it there, guess I should do with an eye check :)

**Comment by andytson** [comment]
closing PR and recreating it with branch on this repo instead

**Pull request closed without merging**